src/style.css

```diff
@@ -0,0 +1,96 @@
+@import url(./style/variables.css);
+@import url(./style/stars.css);
+
+.card {
+  padding: 0 16px 16px;
```

Comment:
add paddings for all sides of the card

```suggestion
  padding: 32px 16px 16px;
```